sdk/Sdk/Sdk.csproj

```diff
@@ -38,6 +38,7 @@
 
     <!-- Marks all packages as 'local only' so they don't end up in the nuspec. -->
     <PackageReference Update="@(PackageReference)" PrivateAssets="All" />
+    <PackageReference Update="SharpZipLib" Version="1.4.1" />
```

Comment:
Can just update line 35 instead.

(Was this VS that inserted the _Update_ item? If so, I may have had a hand in writing that years ago 🤣 )

Reply:
I'm pretty sure this was indeed done by VS. :)